### PR TITLE
[AT-1476] Amazon timestream in PHP example app

### PR DESCRIPTION
**Pull request by dkovalenkoI**
## Summary
Create the started application Amazon Timestream with PHP

## Description
Provide a starter app for PHP to use the Amazon Timestream application and the installation guide.

## Related Issue
[awslabs#42](https://github.com/awslabs/amazon-timestream-tools/issues/42)

### Manual verification:
1. Install PHP based on the guide
2. Install Amazone SDK with Composer
3. Execute the application by `main.php`
4. Verify that database, table and records has been created
5. Verify query results
6. Verify table and database successfully deleted

## Additional Reviewers
@alexeytemnikov

### CODE REVIEW

File: sample_apps/php/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Getting started with Amazon Timestream with PHP
+
+This sample application shows how you can create a database and table, populate the table with sample data records, and run sample queries to jumpstart your evaluation and/or proof-of-concept applications with Amazon Timestream.
```
Comment (forestmvey) [inline sample_apps/php/README.md]: Would be nice if we could split this up into two sentences. Something more like:
```suggestion
The example PHP application gives developers a basic understanding of how to begin using Amazon Timestream with PHP. The application exemplifies the beginning phases of database creation with some sample data population and how to execute queries against a database.
```

File: sample_apps/php/main.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    $reader = createReader();
+    createDatabase($client, DB_NAME);
+    createTable($client, DB_NAME, DB_TABLE_NAME);
+    $records = generateRecords();
+    writeRecords($client, DB_NAME, DB_TABLE_NAME, $records);
+    $results = executeQuery($reader, 'SELECT * FROM '.DB_NAME.'.'.DB_TABLE_NAME);
+    echo $results;
+    deleteTable($client, DB_NAME, DB_TABLE_NAME);
+    deleteDatabase($client, DB_NAME);
+?>
```
Comment (forestmvey) [inline sample_apps/php/main.php]: Ending newline.

**Review by forestmvey — APPROVED** [review]

File: sample_apps/php/README.md (hunk 1 above)
Comment (dkovalenkoI) [reply sample_apps/php/README.md]: done

File: sample_apps/php/main.php (hunk 2 above)
Comment (dkovalenkoI) [reply sample_apps/php/main.php]: done

File: sample_apps/php/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Getting started with Amazon Timestream with PHP
+
+The example PHP application gives developers a basic understanding of how to begin using Amazon Timestream with PHP. The application exemplifies the beginning phases of database creation with some sample data population and how to execute queries against a database.
+
+-----
+## How to use
+### Installation
+#### PHP Installation
+Navigate to [PHP official website](https://www.php.net/downloads.php) and install the stable version of PHP
```
Comment (alexey-temnikov) [inline sample_apps/php/README.md:9]: SPecify version the sample app has been tested with

File: sample_apps/php/main.php [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+    const CLIENT_KEY = 'CLIENT_KEY_OVERRIDE';
+    const CLIENT_SECRET = 'CLIENT_SECRET_OVERRIDE';
+
+    const DB_NAME = 'TEST_PHP';
+    const DB_TABLE_NAME = 'TEST_TABLE_PHP';
+    const MAGNETIC_STORE_RETENTION_PERIOD_IN_DAYS = 1;
+    const MEMORY_STORE_RETENTION_PERIOD_IN_HOURS = 1;
+
+    function createDatabase($client, $dbName) {
```
Comment (alexey-temnikov) [inline sample_apps/php/main.php:14]: Add code comments, sot this code can be used as a tutorial how to get started.

File: sample_apps/php/README.md (hunk 3 above)
Comment (dkovalenkoI) [reply sample_apps/php/README.md:9]: Done, system information has been added

**Pull request merged**